server/routes.js

```diff
@@ -6,14 +6,33 @@
 
 import errors from './components/errors';
 import path from 'path';
+const authMiddlewares=require("./components/auth");
+const redirectUrlFunction = require("./components/auth/redirectUrl");
 
-export default function(app) {
+export default function (app) {
   // Insert routes below
-  app.use('/api/users', require('./api/thing'));
+  app.use('/api/users', require('./api/employee'));
+  //route for logout
+  app.use(authMiddlewares);
+  app.route("/logout")
+    .get((req, res) => {
+      res.clearCookie('Tsms').send("logout successfull");
```

Comment:
Auth.js should have logic for login and logout.

Since, clearing cookie is part of logout, so it belongs to auth.js